src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -151,6 +151,8 @@ export const PTSD_MATCHES = [
   'post-traumatic stress',
 ];
 
+export const MAX_LEVENSHTEIN_DISTANCE = 0.25;
```

Comment:
Might point out that this is a percentage of the whole string length, not an actual distance of 0.25. :wink:

Reply:
Yeah, good point, that name isn't very accurate.